src/App/components/Sidebar/SidebarSearchResults/PositionsSearchResults.tsx

```diff
@@ -94,16 +103,52 @@ export default function PositionsSearchResults(props: propsIF) {
     };
 
     return (
-        <div>
-            <div className={styles.card_title}>My Range Positions</div>
+        <FlexContainer
+            flexDirection='column'
+            justifyContent='center'
+            alignItems='flex-start'
+            gap={8}
+        >
+            <Text fontWeight='500' fontSize='body' color='accent5'>
+                My Range Positions
+            </Text>
             {searchedPositions.length ? (
-                <>
-                    <header className={styles.header}>
-                        <div>Pool</div>
-                        <div>Range</div>
-                        <div>Value</div>
-                    </header>
-                    <ol className={styles.main_result_container}>
+                <FlexContainer flexDirection='column' fullWidth>
+                    <GridContainer
+                        numCols={3}
+                        fullWidth
+                        fontWeight='300'
+                        fontSize='body'
+                        color='text2'
+                        style={{ borderBottom: '1px solid var(--dark3)' }}
+                        padding='0 0 4px 0'
+                    >
+                        <Text
+                            fontWeight='300'
```

Comment:
Same thing again - this style is reused enough to where we should be able to make it a common component.